include/config.h

```diff
@@ -61,5 +66,6 @@ int handler(void *user, const char *section, const char *name,
 /* Utilities */
 void config_split_array_string(char *dest_array[], const char *value, int *len);
 void remove_spaces(char *s);
+int construct_log_filepath();
```

Comment:
Maybe the most correct would be to move `construct_log_filename` to `util.h` to be honest, and keep `construct_log_filepath` to `config.h` since it takes info from the config. Or I can just move both to `log.h` since
both of those functions are related to `log`